packages/smooth_ui_library/lib/navigation/models/smooth_bottom_app_bar_item.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+class SmoothBottomAppBarItem {
```

Comment:
We should be using a new class called `SmoothBottomAppBarItem`, but `BottomNavigationBarItem`.